test/formats/test_mrea.py

```diff
@@ -25,14 +25,8 @@ def test_compare_p2(prime2_asset_manager, mrea_asset_id: AssetId):
     resource = prime2_asset_manager.get_raw_asset(mrea_asset_id)
 
     decoded = Mrea.parse(resource.data, target_game=prime2_asset_manager.target_game)
-    for instance in decoded.all_instances:
```

Comment:
This was intentional, to make sure the MREA is decoded

Reply:
ok I'll put it back then